common/lib/capa/capa/templates/annotationinput.html

```diff
@@ -59,6 +60,5 @@
 </form>
 
 % if msg:
-<span class="message">${msg|n}</span>
+<span class="message">${HTML(msg)}</span>
```

Comment:
Correct. Also, in this case, it sounds like `msg` is sometimes plain text, and not consistently `html_msg`, so this will cover both situations...but we should use the ambiguous name of `msg`.

As an aside, calling HTML() again on HTML won't hurt anything. Calling on plain text will keep it from properly being escaped.